### DISCUSSION TITLE
Nested Golang Struct to database?

### DISCUSSION THREAD

**Discussion by SantoDE** [pocketbase/pocketbase#4664] [Q&A]
Hey there,

I have a nested struct within my CustomModel like that

```
type A struct {
	Extension *B     `db:"-" json:"b"`
	Extension string `db:"b" json:"b"`
}

type B struct {
	something string
}
```

Im trying to figure out how I best lay it out. Do I actually add the struct as a nested field and a separate for the ID, or do I only keep the ID and drop the nested struct? Any best practices?

Thanks!

**Comment by ganigeorgiev** [comment]
I'm not sure I follow. The description is too vague and it is difficult to understand what actually is the question or the desired behavior.

What is the intended purpose of the above struct?
